manifests/pipelines/git-fetch-pipeline.yaml

```diff
@@ -339,21 +377,31 @@ spec:
     - test-model-rest-svc
     - stop-deployment
     workspaces:
-    - name: images-url
+    - name: images_url
       workspace: build-workspace-pv
   - name: skopeo-copy
     params:
-    - name: srcTLSverify
+    - name: SOURCE_IMAGE_URL
+      value: ""
+    - name: DESTINATION_IMAGE_URL
+      value: ""
```

Comment:
I would remove these as they don't add any value and the values are default anyway.